vstools/functions/heuristics.py

```diff
@@ -69,6 +72,7 @@ def try_or_fallback(prop_type: type[SelfPropEnum]) -> SelfPropEnum:
 
 
 def video_resample_heuristics(clip: vs.VideoNode, kwargs: KwargsT | None = None, **fmt_kwargs: Any) -> KwargsT:
+    """Get a kwargs object for a video's heuristics to pass to video plugins."""
```

Comment:
it's specifically for Kernel.resample or resize plugin
also missing all params?

the kwargs are the base out parameters, then you have clip which is where you get the in parameters heuristics from and fmt kwargs is the destination format's replace parameters
said format will be used in combination with the base clip params like resolution to get heuristics for out parameters